### DISCUSSION TITLE
[IDEAS] New LXC Suggestions

### DISCUSSION THREAD

**Discussion by CrazyWolf13** [community-scripts/ProxmoxVE#12] [Ideas]
Hi

As the old repo is now archived, I transferrred some of the most wanted LXC suggestions:

- [x] [Apache Guacamole](https://guacamole.apache.org/)
- [x] [Bookstack](https://www.bookstackapp.com/)
- [x] [Traefik](https://doc.traefik.io/traefik/)
- [x] [Authentik](https://goauthentik.io/) 
- [x] [Photoprism](https://www.photoprism.app/)
- [x] [Ente IO](https://help.ente.io/self-hosting/)
- [x] [Mealie V2](https://mealie.io/)
- [x] [Pterodactyl](https://github.com/pterodactyl)
- [x] [Vikunja](https://vikunja.io/)
- [x] [RustDesk](https://rustdesk.com/)
- [x] [Firefly III](https://www.firefly-iii.org/)
- [x] [Joplin](https://joplinapp.org/)
- [x] [FreshRSS](https://freshrss.org/index.html)
- [x] [Wallabag](https://wallabag.org/)
- [x] [AppFlowy](https://appflowy.io/)
- [x] [Element Synapse](https://github.com/element-hq/synapse)

Source:
- https://github.com/tteck/Proxmox/discussions/categories/ideas
- https://github.com/tteck/Proxmox/discussions/50

**Comment by havardthom** [comment]
Please use the new `Request script` discussion template, so we can track them separately: https://github.com/community-scripts/ProxmoxVE/discussions/new?category=request-script

**Reply by CrazyWolf13** [reply]
Thanks, done :)

**Comment by MickLesk** [comment]
And some scripts are already in the repo :-)
